src/applications/representatives/components/Notifications/Notifications.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export const Notifications = ({ notifications }) => (
+  <div className="vads-u-background-color--white vads-u-padding--2p5 rounded-corners">
+    <a
+      className="view-all-link vads-u-margin-bottom--neg4
+     "
+      href="/poaRequests"
+    >
+      View All
+    </a>
+    <va-table sort-column={1} table-title="POA Requests">
+      <va-table-row slot="headers">
+        <span>Claimant</span>
+        <span>Submitted</span>
+        <span>Accept/ decline</span>
+      </va-table-row>
+      {notifications.map(notification => {
+        return (
+          <va-table-row key={notification.id}>
+            <span>
+              <a href={`/representatives/request-details/${notification.id}`}>
```

Comment:
Sounds good! The mural has it being the drawer for the specific request but as we discussed today who knows if it will be a drawer:
<img width="847" alt="Screenshot 2024-02-21 at 5 11 49 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/a33cf8eb-5d45-4b6e-a986-e307055a50ae">